### ISSUE TITLE
[bitnami/elasticsearch] README: description for 'security.existingSecret' is an incomplete sentence and is missing details

### ISSUE THREAD

**Issue by johanneskastl** [bitnami/charts#20545]
### Name and Version

current state in main

### What architecture are you using?

None

### What steps will reproduce the bug?

(I found no way to not make this a tech-issue, sorry)

Check the following line:
https://github.com/bitnami/charts/blob/main/bitnami/elasticsearch/README.md?plain=1#L121

The sentence is incomplete, it is probably missing "and the name of the admin user" or similar at the end.

And it would be nice to have a more detailed description on what needs to be in that secret. How should the keys be named? Which keys need to be present?

### What do you see instead?

```
| `security.existingSecret`                  | Name of the existing secret containing the Elasticsearch password and                                                                               | `""`                            |
```

Notice that the sentence stops at `and`.

**Comment by carrodher** [comment]
Thank you for bringing this issue to our attention. We appreciate your involvement! If you're interested in contributing a solution, we welcome you to create a pull request. The Bitnami team is excited to review your submission and offer feedback. You can find the contributing guidelines [here](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md).

Your contribution will greatly benefit the community. Feel free to reach out if you have any questions or need assistance.

**Comment by johanneskastl** [comment]
@carrodher Thanks for the reply, but I am lacking the insight into the inner workings of elasticsearch, so I do not know what exactly should be in that secret, what names should be used, etc. If there is documentation on this I can of course come up with a PR for this...

**Comment by fevisera** [comment]
Hi @johanneskastl,

I created #20631 to address the Readme and added some comments related to the usage of that parameter. 
Kindly review it and add comments if you believe there are other aspects related to this issue that should be addressed in that PR. This issue will be automatically closed once the PR is merged.

Thanks!

**Comment by johanneskastl** [comment]
Thanks for caring, I commented the PR with my thoughts
